oak_abi/proto/label.proto

```diff
@@ -82,3 +83,8 @@ message TlsEndpointTag {
   // using the set of Certificate Authorities (CA) that are available to it.
   string authority = 1;
 }
+
+// Message representing top-secret. Declaring this as an enum results in conflicting `derive` 
+// rules between the ones that Prost automatically generates for enums, and the ones that are
```

Comment:
I am not sure I understand the sentence about enum, could you clarify? Are you talking about a Rust enum?

Reply:
I wanted to have it as an enum. Prost converts this to a Rust enum, but then there is conflict in the derived annotation: 
```
error[E0119]: conflicting implementations of trait `std::cmp::Eq` for type `proto::oak::label::TopTag`:
   --> /opt/my-project/oak_runtime/target/debug/build/oak_abi-941eb48ba258a197/out/oak.label.rs:111:10
    |
111 | #[derive(Eq, Hash)]
    |          ^^ conflicting implementation for `proto::oak::label::TopTag`
...
114 | #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, ::prost::Enumeration)]
    |                                         -- first implementation here
    |
    = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

It is related to this issue: https://github.com/danburkert/prost/issues/332.

Removed it from the comment.